helios/pipeViewer/pipe_view/model/database.py

```diff
@@ -12,36 +12,11 @@
 from .location_manager import LocationManager
 from .clock_manager import ClockManager
 
-# Import Argos transaction database module from SPARTA
-__MODULE_ENV_VAR_NAME = 'TRANSACTIONDB_MODULE_DIR'
-env_var = os.environ.get(__MODULE_ENV_VAR_NAME)
-if env_var is None:
-    # Try to find the transaction module in the Helios release dir
-    added_path = os.path.dirname(__file__) + \
-        "/../../../../release/helios/pipeViewer/transactiondb/lib"
-    added_path = os.path.abspath(added_path)
-    if not os.path.isdir(added_path):
-        error('Argos cannot find the transactiondb directory: %s', added_path)
-        sys.exit(1)
-else:
-    added_path = os.environ.get(__MODULE_ENV_VAR_NAME, os.getcwd())
```

Comment:
@bdutro , with Peter's changes (which includes a proper install of Argos/Sparta), this goes away.  @MichaelSchoenfelder-SiV, we should chat about how to take advantage of this.

Reply:
This should be fine, worst case we can always tweak `PYTHONPATH` with a wrapper script.

Reply:
Nevermind, just saw that we're doing a relative import now. That should make it even easier.